src/ufo.rs

```diff
@@ -114,22 +177,32 @@ impl Default for MetaInfo {
 }
 
 impl Ufo {
-    /// Crate a new `Ufo`.
+    /// Create a new `Ufo`.
     pub fn new() -> Self {
         Ufo::default()
     }
 
+    /// Create a new `Ufo` only with certain fields
+    pub fn with_fields(data_request: DataRequest) -> Self {
+        let mut ufo = Self::new();
+        ufo.data_request = data_request;
+        ufo
+    }
+
     /// Attempt to load a font object from a file. `path` must point to
     /// a directory with the structure described in [v3 of the Unified Font Object][v3]
     /// spec.
     ///
     /// [v3]: http://unifiedfontobject.org/versions/ufo3/
     pub fn load<P: AsRef<Path>>(path: P) -> Result<Ufo, Error> {
+        Self::new().load_ufo(path)
+    }
+
+    pub fn load_ufo<P: AsRef<Path>>(&self, path: P) -> Result<Ufo, Error> {
```

Comment:
I'm feeling a bit thick, can you help me understand this change? it seems like we now have two differently named functions that do the same thing?

In any case, if we're adding new public API I'd like at least a short doc comment. :)

Reply:
It assures that old code works unchanged, as we previously did not require Ufo type be initialized before calling `load()`. So, that is maintained, with us now doing the initialization so API consumers won't notice anything has changed.

Reply:
Following up. :-)

Reply:
makes sense!